daemon/cmd/daemon_main.go

```diff
@@ -1596,8 +1596,11 @@ func initEnv(vp *viper.Viper) {
 	}
 }
 
-func (d *Daemon) initKVStore(resolver *dial.ServiceResolver) {
+func (d *Daemon) initKVStore(svcResolver *dial.ServiceResolver, cmResolver *dial.ClustermeshResolver) {
```

Comment:
This etcd client is used to access the Cilium's KVStore (when running in KVStore mode), no need to configure the extra resolver.